File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples20.vb
' <snippetlinqexamples20>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_gt1 = From c In svcContext.ContactSet _
                  Where c.Anniversary > New Date(2010, 2, 5) _
                  Select New With {Key c.FirstName,
                                   Key c.LastName,
                                   Key c.Address1_City}

  For Each c In query_gt1
   Console.WriteLine(c.FirstName &amp; " " &amp; c.LastName _
                     &amp; " " &amp; c.Address1_City)
  Next c
 End Using

' </snippetlinqexamples20>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples20.cs
// <snippetlinqexamples20>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_gt1 = from c in svcContext.ContactSet
                  where c.Anniversary > new DateTime(2010, 2, 5)
                  select new
                  {
                   c.FirstName,
                   c.LastName,
                   c.Address1_City
                  };

  foreach (var c in query_gt1)
  {
   System.Console.WriteLine(c.FirstName + " " +
    c.LastName + " " + c.Address1_City);
  }
 }

// </snippetlinqexamples20>